server/config.js

```diff
@@ -99,23 +99,221 @@ module.exports = {
       getEnv('APPROVED_VISITORS_FEATURE_ENABLED', 'false') === 'true',
```

Comment:
Still not sure whether to keep these flags as the new features would control this.